packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -303,10 +309,10 @@ class Frame extends React.Component {
     } else {
       // Allows time for state update to render select menu DOM before shifting focus to it
       setTimeout(() => {
-        if (document.querySelector(this.selectMenu)) {
-          document.querySelector(this.selectMenu).focus();
+        if (this.selectMenu) {
+          this.selectMenu.focus();
         }
-      }, 10);
+      }, 100);
```

Comment:
curious, how did you land on 100?

Reply:
hmm for reuse, can we extract this setTimeout and focus logic into a function? maybe we call it `shiftFocusToMenu` or something

Reply:
Moved setTimeout and focus to a function in FrameUtil, 875db90.
Stephen had done an initial investigation and suggested to go for a 100ms value [here](https://github.com/cerner/terra-core/issues/3035#issuecomment-644888940).